add article that links to statistics courses (#235)
inbo/tutorials: content/articles/inbo_stats_courses/index.md

date: 2021-02-24
categories: ["r", "statistics"]
tags: ["r", "INLA", "mixed model", "uncertainty", "explorative data analysis", "analysis of variance", "generalized linear regression", "multivariate statistics"]
output: 
    md_document:
        preserve_yaml: true
        variant: gfm
---

During the course of the years, we have taught many statistics courses at the Research Institute for Nature and Forest (INBO), both for scientists and technicians.
Most of these courses have been developed and taught in-house.
Some (mostly advanced) courses were taught in-house by external teachers.

While we do plan to periodically repeat these courses for new employees and as a refresher, the course material can always be consulted and used for self-teaching or as a reference.

We chose to maintain the links to googledrive folders with the course material on our intranet and therefore limit ourselves here to just point you to that link.
You can find the overview [here](https://sites.google.com/a/inbo.be/intranet/ondersteuning/onderzoek/bmk/lessenreeks-en-tutorials).
This course material is only accessible for INBO employees.

> Do note, however, that more recently developed in-house statistics courses will most likely end-up being part of this public website.

Reviewer: ```suggestion
Do note, however, that more recently developed in-house statistics courses will most likely end up being part of the public tutorials website you are visiting.
```

---
title: "Statistics courses @ INBO!"
description: "Links to course material of statistics courses taught at INBO (only accessible for INBO employees)"
date: 2021-02-24
categories: ["r", "statistics"]
tags: ["r", "INLA", "mixed model", "uncertainty", "explorative data analysis", "analysis of variance", "generalized linear regression", "multivariate statistics"]
output: 
    md_document:
        preserve_yaml: true
        variant: gfm
---

During the course of the years, we have taught many statistics courses at the Research Institute for Nature and Forest (INBO), both for scientists and technicians.
Most of these courses have been developed and taught in-house.

> Some (mostly advanced) courses were taught in-house by external teachers.

Reviewer 2: ```suggestion
Some (mostly advanced) courses were taught in-house by external teachers.
```
I'm not sure if we should mention external courses here as well...

Author: I think that is OK. I will leave it as is.

While we do plan to periodically repeat these courses for new employees and as a refresher, the course material can always be consulted and used for self-teaching or as a reference.

We chose to maintain the links to googledrive folders with the course material on our intranet and therefore limit ourselves here to just point you to that link.
You can find the overview [here](https://sites.google.com/a/inbo.be/intranet/ondersteuning/onderzoek/bmk/lessenreeks-en-tutorials).
This course material is only accessible for INBO employees.

Do note, however, that more recently developed in-house statistics courses will most likely end-up being part of this public website.
An example is the [follow-up INLA workshop](https://inbo.github.io/tutorials/tutorials/r_inla/).

> Moreover, we have several short tutorials about very specific statistical topics.

Reviewer 2: ```suggestion
Moreover, we have several short tutorials on very specific statistical topics.
```
I'm not sure which is the best, but this sounds better for me.
